public/config/configuration.json

```diff
@@ -2,7 +2,6 @@
   "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "AUTH_URL": "https://dev.bcregistry.ca/cooperatives/auth/",
   "SIGNIN_URL": "https://dev.bcregistry.ca/cooperatives/auth/signin/bcsc/",
-  "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
```

Comment:
You replaced this key with the new one, but this project still needs this value. See https://github.com/bcgov/bcrs-business-create-ui/blob/b6da247b368cf2b35aa1f0bed5ecc76c9ea68081/src/utils/auth-helpers.ts#L86

Reply:
done